Exercises/Modules/Chapter 19/Soln19_02/Sort.cppm

```diff
@@ -0,0 +1,49 @@
+export module sort;
+
+import <utility>;    // for std::swap()
+import <vector>;
+
+// The primary, exported sort() template with two parameters calls 
+// this internal sort() template with four parameters
+template<typename T, typename Compare>
```

Comment:
Why not put the export here? As the comment says "exported sort()".

Reply:
It's the other sort template that is exported, not this recursive one.

Reply:
Right, I read the sentence too quickly :(